department-of-veterans-affairs/vets-website
# 86798 Remove includeToxicExposure flag

## Summary
- Remove any functionality or references to the no longer used `includeToxicExposure` flag

team: @department-of-veterans-affairs/dbex-trex 
toggle cleanup: department-of-veterans-affairs/va.gov-team#65089

## Related issue(s)
department-of-veterans-affairs/va.gov-team#86798

## Testing done
- Unit and e2e tests updated

**Manual test setup**
**1. TE pages continue to display for new applications (started version 2022)**
1.1 enable only TE toggle `disability_526_toxic_exposure`. Do not enable the IPF toggle.
1.2 start a new 526 form, filling out all required fields until you get to the claim-type page. Select 'A new condition that’s connected to my service'
1.3 **Verify**: make sure the /toxic-exposure/conditions page displays and you are able to submit a claim with any of the TE pages filled out.
1.4 **Verify**: make sure startedFormVersion: "2022" is included in the submit payload

**2. TE pages continue to display for IPF forms with logic enabled (started version 2019)**
2.1 disable both TE toggles
2.2 start a new claim with new conditions. Only go a couple pages in and then click 'Finish this application later' link to save progress
2.3 enable ipf toggle `disability_526_toxic_exposure_ipf`
2.4 Hard refresh the page. Resume filling out the application for all required fields. 
2.5 On the claim-type page, select 'A new condition that’s connected to my service '.
2.6 **Verify**: make sure the /toxic-exposure/conditions page displays and you are able to submit a claim with any of the TE pages filled out.
2.4 **Verify**: make sure startedFormVersion: "2019" is included in the submit payload

**3. TE pages do not display for IPF forms not enabled (not the chosen ones)**
3.1 disable both TE toggles
3.2 start a new 526 form, filling out all required fields until you get to the claim-type page. Select 'A new condition that’s connected to my service'
3.3 save progress by clicking 'Finish this applicaiton later'
3.4 optionally enable the new applications toggle "disability_526_toxic_exposure"
3.5 **Verify**: leave the form or sign out and come back. make sure TE pages are still not visible for the IPF.

**4. TE pages do not display for new applications not enabled (not the chosen ones)**
4.1 disable toggle `disability_526_toxic_exposure`
4.2 start a new 526 form, filling out all required fields until you get to the claim-type page. Select 'A new condition that’s connected to my service'
4.3 **Verify**: make sure TE pages are still not visible.

## Screenshots
n/a, no visual changes

## What areas of the site does it impact?
526ez

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
